packages/api/src/graphql/query.ts

```diff
@@ -798,14 +841,15 @@ export const GraphQLPublicQuery = new GraphQLObjectType<undefined, Context>({
     article: {
       type: GraphQLPublicArticle,
       args: {id: {type: GraphQLID}},
-      async resolve(root, {id}, {session, loaders}) {
+      async resolve(root, {id}, {session, loaders, dbAdapter}) {
         const article = await loaders.publicArticles.load(id)
+        const articleComments = await dbAdapter.comment.getPublicComments([id])
```

Comment:
this is something that should happen in the resolve function of GraphQLPublicArticle

Reply:
fixed